### DISCUSSION TITLE
Docker Run horaedb mount data problem

### DISCUSSION THREAD

**Discussion by yutianfanxing** [CeresDB/ceresdb#1551] [Q&A]
When I use docker to start horaeDB like this:
![image](https://github.com/user-attachments/assets/a689a4b1-9ae0-43e6-9365-0b811e9ee3b1)
and I use docker inspect horaedb-server:
![image](https://github.com/user-attachments/assets/47cf99b9-d60e-43b5-bc20-c7e45a457321)
Now, I have inserted 10 millions rows into one table, but I find my `/data/horaedb` is empty.
![image](https://github.com/user-attachments/assets/8fc3456b-8c6f-4bcc-9a5b-00f1a253cf20)

**Comment by yutianfanxing** [comment]
I find this in container:
![image](https://github.com/user-attachments/assets/dc60caa3-e914-4820-aace-c9160cdb26d0)
Is this directory storing the horaedb's data?

**Reply by jiacai2050** [reply]
Sorry for delayed response.


> Is this directory storing the horaedb's data?


Yes, the server use this config file: https://github.com/apache/horaedb/blob/main/docs/minimal.toml
